conda/exceptions.py

```diff
@@ -617,6 +617,13 @@ def __init__(self, packages, channel_urls=()):
 
             and use the search bar at the top of the page.
             """)
+            from .base.context import context
+
```

Comment:
Let's add a test to this, could go right inside 

```def test_use_only_tar_bz2(platform=OVERRIDE_PLATFORM):
    channel = Channel(join(CHANNEL_DIR, platform))
    SubdirData.clear_cached_local_channel_data()
    with env_var("CONDA_USE_ONLY_TAR_BZ2", True, stack_callback=conda_tests_ctxt_mgmt_def_pol):
        sd = SubdirData(channel)
        precs = tuple(sd.query("zlib"))
        assert precs[0].fn.endswith(".tar.bz2")
```

Only need to raise the exception, and check the message to get code coverage. No need to really "not find" a package.

Reply:
I think this hasn't reached the level of documentation, but it'd be worth it

Reply:
Added test in `test_exceptions.py`.